pkg/pool-stable/contracts/ComposableStablePool.sol

```diff
@@ -1013,6 +1013,24 @@ contract ComposableStablePool is
         _updatePostJoinExit(currentAmp, currentInvariant);
     }
 
+    function _onDisableRecoveryMode() internal override {
+        // Enabling recovery mode forcefully sets all protocol fee percentages to zero, increasing the return value of
+        // `getRate()` and effectively forfeiting due protocol fees.
```

Comment:
```suggestion
        // Enabling recovery mode short-circuits protocol fee computations so that all percentages are
        // effectively zero, increasing the return value of `getRate()` and effectively forfeiting due protocol fees.
```
The original wording sounds like there is code running that overwrites rates.

Reply:
The protocol fee computations still run, they just run with 0 values, "short-circuits protocol fee computations" sounds more like it prevents the computations altogether.

What about something along the lines of below, we also try to clarify why getRate increases:

// Enabling recovery mode short-circuits protocol fee percentage retrieval, returning a zero percentage for all fee types. This effectively forfeits any due protocol fees (unminted BPTs), reducing the actualTotalSupply and increasing the value returned by `getRate()`.